client/src/app/containers/Login/index.jsx

```diff
@@ -82,7 +82,7 @@ const LoginForm = () => {
           </Button>
         </form>
         <Link href='/register' variant='body2'>
-          Don't have an account? Register here
+          Don&apos;t have an account? Register here
```

Comment:
why not wrap the string in `{}`? any particular reason?

Reply:
This was a change to make the linter happy - it was working fine before too. But a JS string is probably a better call